client/src/store/new-request-module.ts

```diff
@@ -1995,9 +1995,10 @@ export class NewRequestModule extends VuexModule {
   @Action
   async addRequestActionComment (data) {
     try {
-      let requestAction = this.requestActionOriginal || this.request_action_cd
-      let { shortDesc } = this.requestActions.find(request => request.value === requestAction)
```

Comment:
This was causing "addRequestActionComment() = TypeError: Right side of assignment cannot be destructured."